README.md

```diff
@@ -98,6 +91,77 @@ sync(config, callback)
 | config.scmUrl | String | Yes | Source Code URL for the application |
 | callback | Function | Yes | Callback function|
 
+### Job Model
+```js
+'use strict';
+const Model = require('screwdriver-models');
+
+module.exports = (datastore) => ({
+    method: 'GET',
+    path: '/jobs',
+    config: {
+        description: 'Get jobs with pagination',
+        notes: 'Returns all job records',
+        tags: ['api', 'jobs'],
+        handler: (request, reply) => {
+            const Job = new Model.Job(datastore);
+            Job.list(request.query, reply);
+        }
+    }
+});
+```
+
+#### Create
+Create a new job
+```
+create(config, callback)
+```
+
+| Parameter        | Type  |  Description |
+| :-------------   | :---- | :-------------|
+| config        | Object | Configuration Object |
+| config.pipelineId | String | The pipelineId that the job belongs to |
+| config.name | String | The name of the job |
+| callback | Function | Callback function|
+
+#### Get
+Get a job based on id
+```
+get(id, callback)
+```
+
+| Parameter        | Type  |  Description |
+| :-------------   | :---- | :-------------|
+| id | String | The unique ID for the job |
+| callback | Function | Callback function|
```

Comment:
The callback isn't just a notification. The callback also returns back data. We should definitely document what gets passed to the callback function. Should we document what the return data looks like?